extensions/CESIUM_binary_glTF/README.md

```diff
@@ -41,7 +42,7 @@ Given an `arrayBuffer` with Binary glTF, Listing 1 shows how to parse the header
 
 **Listing 1**: Parsing Binary glTF.  This uses a `TextDecoder` wrapper in Cesium, [`getStringFromTypedArray`](https://github.com/AnalyticalGraphicsInc/cesium/blob/bgltf/Source/Core/getStringFromTypedArray.js).
 ```javascript
-var sizeOfUnit32 = Uint32Array.BYTES_PER_ELEMENT;
```

Comment:
Good eye, thanks.

Could you also please open a small pull request into Cesium to fix this in https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Model.js?

Run the tests before opening the pull request.